packages/itwinui-css/src/alert/alert.scss

```diff
@@ -72,7 +72,7 @@
     @include iui-anchor-status($statusColor);
   }
 
-  #{$rootSelector}-button {
+  #{$rootSelector}-button:focus-visible {
```

Comment:
[Test comment] What is the specificity of this?

Reply:
it's [0, 3, 0]